conf/messages.cy

```diff
@@ -139,7 +139,7 @@ status.removed = wedi’i ddileu
 status.paid = wedi’i dalu
 status.cancelled = wed’i ganslo
 status.reinstated = wedi’i ailosod
-status.estimated = wedi’i amcangyfrifo
+status.estimate = wedi’i amcangyfrif
```

Comment:
I think the Welsh is 'amcangyfrif' for' estimate'
<img width="414" alt="Screenshot 2022-10-10 at 12 54 26" src="https://user-images.githubusercontent.com/86231805/194860614-fb512ff2-b5f9-4af3-8a06-bba321e58bb9.png">